README.md

```diff
@@ -1,43 +1,71 @@
-# onnx-cpu
+# `onnx-cpu` modular resource
 
-## Build it
+This module allows you to run ONNX models with the Viam Vision Service on Android.
```

Comment:
```suggestion
This module allows you to run ONNX models with the Viam `mlmodel` vision service on Android.
```
Makes it seem like there is only one vision service otherwise